events/forms.py

```diff
@@ -0,0 +1,32 @@
+from django import forms
+from django.forms import ModelForm
```

Comment:
because you use it only once and you already imported the forms in the first line
I think you can get rid of this line and in line # 8 do it like:
 class EventCreationForm(forms.ModelForm):

Reply:
done